### PR TITLE
EL7 migration from python 2 to python 3

### PR DESCRIPTION
**Pull request by amanda11**
Documentation changes for python3 use on EL7 distributions

Part of StackStorm/discussions#54

### CODE REVIEW

File: docs/source/upgrade_notes.rst [hunk 1]
```diff
@@ -3,6 +3,13 @@
 Upgrade Notes
 =============
 
+.. _ref-upgrade-notes-v3-4dev:
+
+|st2| v3.4dev
```
Comment (armab) [inline docs/source/upgrade_notes.rst:8]:
```suggestion
|st2| v3.4
```
As we're preparing the documentation for upcoming 3.4 release. 3.4dev is irrelevant to the users.

File: docs/source/install/upgrades.rst [hunk 2]
```diff
@@ -167,6 +167,38 @@ The following sections call out the migration scripts that need to be run when u
 respective version. If you are upgrading across multiple versions, make sure you run the scripts for
 any skipped versions:
 
+v3.4dev
```
Comment (armab) [inline docs/source/install/upgrades.rst:170]:
```suggestion
v3.4d
```

File: docs/source/install/rhel7.rst [hunk 3]
```diff
@@ -34,6 +39,9 @@ you may want to tweak them according to your security practices.
     # SELINUX management tools, not available for some minimal installations
     sudo yum install -y policycoreutils-python
 
+    # SELINUX python3 rules
+    sudo yum install -y libselinux-python3
+
     # Allow network access for nginx
     sudo setsebool -P httpd_can_network_connect 1
```
Comment (armab) [inline docs/source/install/rhel7.rst:46]: Following the discussion in https://github.com/StackStorm/ansible-st2/pull/278 we discovered that `libselinux` is the requirement for specific Ansible module, but not for the st2 installation itself.

I also don't see it between the requirements for EL8 installation:
https://github.com/StackStorm/st2-packages/blob/b3e6674de987198b88f0fe3a3caf332e5037059c/scripts/st2bootstrap-el8.sh#L478-L485

With that, looks like we can remove this dependency, not sure though where it came from as `setsebool` tool required `policycoreutils-python` package only.

Comment (amanda11) [reply docs/source/install/rhel7.rst:46]: Snap - was just about to remove it!

File: docs/source/install/upgrades.rst [hunk 4]
```diff
@@ -167,6 +167,38 @@ The following sections call out the migration scripts that need to be run when u
...
+*  |st2| now uses python 3 on RHEL/CentOS 7. Therefore any packs that only support python 2 will need to be upgraded to python 3.
+
+
+* If SELINUX is enabled, install the python 3 SELinux rules with:
+
+  .. sourcecode:: bash
+
+    sudo yum install -y libselinux-python3
+
+* Ensure python3-devel can be installed from an enabled repository:
```
Comment (armab) [inline docs/source/install/upgrades.rst:182]: Is the `python3-devel` absence only an issue for RHEL systems?
If so, worth mentioning that so CentOS users don't need to worry about this instruction.

File: docs/source/install/upgrades.rst [hunk 5]
```diff
@@ -167,6 +167,38 @@ The following sections call out the migration scripts that need to be run when u
...
+''''
+
+*  |st2| now uses python 3 on RHEL/CentOS 7. Therefore any packs that only support python 2 will need to be upgraded to python 3.
+
+
+* If SELINUX is enabled, install the python 3 SELinux rules with:
+
+  .. sourcecode:: bash
+
+    sudo yum install -y libselinux-python3
```
Comment (armab) [inline docs/source/install/upgrades.rst:180]: `libselinux-python3` could be removed here too.

**Review by armab — APPROVED** [review]
:+1:

**Pull request merged**